### DISCUSSION TITLE
install VS Code Jupyter extension in an offline computer

### DISCUSSION THREAD

**Discussion by arif580** [microsoft/vscode-jupyter#14047] [General]
Hello,
I am trying to download vs code jupyter extension from the market  place

https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter&ssr=false#qna
I am transfering the ms-toolsai.jupyter-2023.7.1002152029@win32-x64.vsix file into a remote computer running windows server2019.
unfortunately I am getting the message that it is not compatible and I can not find kernel.

![jupyter_add in download](https://github.com/microsoft/vscode-jupyter/assets/80361793/f6c6dfd9-017a-42b9-a778-afb3f453b67c)

![image](https://github.com/microsoft/vscode-jupyter/assets/80361793/975d3f4b-1dbd-4736-83bf-e68172a74821)

![image](https://github.com/microsoft/vscode-jupyter/assets/80361793/a2cf9ba2-7dd5-46a6-9101-2f26afc654e7)

it is woking fine in a comptuer that has internet connection. Where can I find the correct compatible jupyter extension vsix file for the offline comptuer. Is there any specific protocol to install this? Thank you.

**Comment by Deasetopia** [comment]
Having similar issues with resolution of versions, even listing compatible versions of VS Code would be helpful.

**Comment by QiDiSong** [comment]
I have this problem also, how can i find the correct offline jupyter extension.
Thanks a lot!

**Comment by rmgouldny** [comment]
I did not see this older discussion when I noted the same issue with a guess at the root cause.  I believe you will find the workaround I mention in the link ( https://github.com/microsoft/vscode-jupyter/discussions/15182 ) to my post useful.

**Comment by Barry1915** [comment]
> Hello, I am trying to download vs code jupyter extension from the market place
> 
> https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter&ssr=false#qna I am transfering the [ms-toolsai.jupyter-2023.7.1002152029@win32-x64.vsix](mailto:ms-toolsai.jupyter-2023.7.1002152029@win32-x64.vsix) file into a remote computer running windows server2019. unfortunately I am getting the message that it is not compatible and I can not find kernel.
> 
> ![jupyter_add in download](https://private-user-images.githubusercontent.com/80361793/257964285-f6c6dfd9-017a-42b9-a778-afb3f453b67c.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjcxNTI0NDksIm5iZiI6MTc2NzE1MjE0OSwicGF0aCI6Ii84MDM2MTc5My8yNTc5NjQyODUtZjZjNmRmZDktMDE3YS00MmI5LWE3NzgtYWZiM2Y0NTNiNjdjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTEyMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMjMxVDAzMzU0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcxNzZiYjBmYjdmOTg2YTI3MWNmMmIzNTJlZDgyZjE3OWU1YzMwM2YwZDVmYjgyYWIyYjlmNWExNjhjYjkyZTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Z9Wr0yvJKz4WqeT8sKy8ERg9rRFR24ABJtx8XaM9MG0)
> 
> ![image](https://private-user-images.githubusercontent.com/80361793/257964678-975d3f4b-1dbd-4736-83bf-e68172a74821.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjcxNTI0NDksIm5iZiI6MTc2NzE1MjE0OSwicGF0aCI6Ii84MDM2MTc5My8yNTc5NjQ2NzgtOTc1ZDNmNGItMWRiZC00NzM2LTgzYmYtZTY4MTcyYTc0ODIxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTEyMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMjMxVDAzMzU0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVkOWVjMGFjOTA4ZDQxYmY0NTJmYWNmNjk5YjViNTQxZWU0MzY1MDA4ZmE3ZWM0MGY3YWQ2OTgyOWZkNjFhY2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.TIoaztOrZ6xGSI-xJoyyzbflbpOckSrt9R3w__dKKpM)
> 
> ![image](https://private-user-images.githubusercontent.com/80361793/257964791-a2cf9ba2-7dd5-46a6-9101-2f26afc654e7.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjcxNTI0NDksIm5iZiI6MTc2NzE1MjE0OSwicGF0aCI6Ii84MDM2MTc5My8yNTc5NjQ3OTEtYTJjZjliYTItN2RkNS00NmE2LTkxMDEtMmYyNmFmYzY1NGU3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTEyMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMjMxVDAzMzU0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYzYzg1NDdmMmMxMjIyYjUwNTRjMDZhZDZkMjA2ZTFkNjk3OTgxZThlNmY5NDliNzE0YTRhNDk4N2QyNjJjZWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.S2Lqu5xOHPvWhoAy-fK-4v6IoEjpvzcNXFgYctvkUEk)
> 
> it is woking fine in a comptuer that has internet connection. Where can I find the correct compatible jupyter extension vsix file for the offline comptuer. Is there any specific protocol to install this? Thank you.

<img width="1936" height="1300" alt="image" src="https://github.com/user-attachments/assets/fdfcff01-f779-4735-b1d2-304e11ebb956" />
